specification/security/resource-manager/Microsoft.Security/preview/2023-07-01-preview/examples/ApplicationMappingRules/GetSecurityConnectorApplicationMappingRule_example.json

```diff
@@ -0,0 +1,32 @@
+{
+  "parameters": {
+    "api-version": "2023-07-01-preview",
+    "scope": "subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/resourceGroups/gcpResourceGroup/providers/Microsoft.Security/securityConnectors/gcpconnector",
+    "ruleId": "ad9a8e26-29d9-4829-bb30-e597a58cdbb8"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/resourceGroups/gcpResourceGroup/providers/Microsoft.Security/applicationMappingRules/ad9a8e26-29d9-4829-bb30-e597a58cdbb8",
+        "name": "ad9a8e26-29d9-4829-bb30-e597a58cdbb8",
+        "type": "Microsoft.Security/applicationMappingRules",
+        "properties": {
+          "tenantId": "f0b6d37b-e4bc-4719-9291-c066c3194f23",
+          "environmentId": "subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/resourceGroups/gcpResourceGroup",
```

Comment:
I'm not sure if the resource group is a valid environment, could you please validate it with Roi / Governance stream?